src/views/Tasks/index.vue

```diff
@@ -1,14 +1,239 @@
 <template>
-  <v-container>
-    <h1 class="display-3 grey--text my-5">Tasks</h1>
-    <br />
-    <br />
-    <div class="title grey--text">
-      Follow the guidelines below while designing your API
-    </div>
-  </v-container>
-</template>
+    <v-container>
+      <h1 class="display-3 grey--text my-5">Tasks</h1>
+      <br />
+      <br />
+      <div class="title grey--text">
+        Follow the guidelines below while designing your API
+      </div>
+      <br />
+      <br />
+      
+      <span class="border-bottom border-dark"></span>
+  
+      <div id="table-of-contents"> 
+      <h3 class="title">Table of contents</h3>
+        <v-divider></v-divider>
+        <ul>
+          <li class="title"><a style="text-decoration:none" href="#liveness">Liveness</a></li>
+          <li class="title"><a style="text-decoration:none" href="#create-article">Create articles</a></li>
+          <li class="title"><a style="text-decoration:none" href="#get-all-articles">Get all articles</a></li>
+          <li class="title"><a style="text-decoration:none" href="#update-article">Update article</a></li>
+          <li class="title"><a style="text-decoration:none" href="#delete-article">Delete article</a></li>
+        </ul>
+        <br />
+      </div>
 
-<script>
-export default {};
-</script>
+      <div id="liveness">
+        <h3 class="display-1">Liveness</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>
+          A GET on the 
+          <code>"/status"</code> 
+           route is expected to return the following JSON.
+        </p>
+        <br />
+  
+        <v-card >
+          <v-card-subtitle>
+            GET /status response:
+          </v-card-subtitle>
+          <v-card-text>
+            <pre>
+  {
+    <span class="red--text">"status"</span> : <span class="red--text">"Up"</span>
+  }
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+      </div>
+  
+     <div id="articles">
+        <h3 class="display-1">Articles</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>
+          The website's main features are articles. The user should be able to create, store, edit, view, and delete articles item.
+          Therefore, as a developer, you need to implement the backend functionality to support the following http requests:  
+        </p>
+  
+        <br />
+  
+        <div id="create-article">
+        <h3 class=title>Create articles</h3>
+        <v-divider></v-divider>
+        <br />
+  
+        <p>A POST on route <code>"/articles"</code> should create a new entry for an article in the database. If the submitted article is invalid (e.g. missing title or author), the route should return status message 400. If POST is successful, it should return status message 200.
```

Comment:
1. Replace "status message" with "status code"
2. restructure the sentence as follows: "Return a status code 400 if any of the parameters is missing from the body" instead of "If the submitted article ... message 400"
3. "...respond with a status 200 with the newly created article"